.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -60,6 +66,5 @@ body:
       - Required, unless the reproduction steps are trivial and don't require any project files to be followed. In this case, write "N/A" in the field.
       - Drag and drop a ZIP archive to upload it. **Do not select another field until the project is done uploading.**
       - **Note for C# users:** If your issue is *not* C#-specific, please upload a minimal reproduction project written in GDScript. This will make it easier for contributors to reproduce the issue locally as not everyone has a .NET setup available.
-      - **If you've been asked by a maintainer to upload a minimal reproduction project, you *must* do so within 7 days.** Otherwise, your bug report will be closed as it'll be considered too difficult to diagnose.
```

Comment:
We discussed removing this line a few weeks ago, as it sounds much stricter than we actually are. MRPs are a best effort and the lack thereof isn't reason for closing an issue, unless we estimate that there's really nothing of value in the bug report.

Reply:
Might word it more like "If the steps are not trivial, or others are unable to reproduce your bug from the reproduction steps, not providing a minimal reproduction project might lead to the issue being closed, as nothing can be done to confirm the bug. It also helps confirming that a bug has been solved."

Actually I think making a point about how the MRP isn't *just* about confirming the bug is important, making it clear it's also for confirming a fix is helpful as some don't realize it and think it's just to "prove" their bug exists.

Reply:
Added some stuff, WDYT?

Reply:
Looks great!